api/python/provisioner/api_spec.yaml

```diff
@@ -115,6 +115,8 @@ salt:
   type: SaltClient
 generate_roster:
   type: GenerateRoster
+refresh_machine_id:
+  type: RefreshMachineId
```

Comment:
Rename this to reset as we are actually re-setting the id.

Reply:
comment is addressed